infra/relayer_addresses.json

```diff
@@ -41,14 +41,20 @@
     "ghs_usd": "0x1485C0E710ff9EF059834F42b312F10e7af823bd",
     "gbp_usd": "0x215d3ba962597DeFb38Da439ED4dB8E8a63e409a",
     "zar_usd": "0x4FF9042aF59AF2B507b9423bE385f664FF87F7af",
+    "chf_usd": "0x09B310c2D4b0CDCE563762C0e3992e352Cacdda6",
```

Comment:
this is the address for celo_chf i think

Reply:
updated